### PR TITLE
uniq

### PR DESCRIPTION
**Pull request by kris-nova**

### CODE REVIEW

File: pull-request-suggester/processor/processor.go [hunk 1]
```diff
@@ -75,7 +75,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
 			continue
 		}
 		// Hooray we have a result back from the model
+		var cache []string
 		for _, f := range response.FileNameCommitIDPositions {
+			uniqkey := fmt.Sprintf("%s%s%d", f.CommitID, f.FileName, f.Position)
+			if !in(cache, uniqkey) {
+				cache = append(cache, uniqkey)
+			} else {
```
Comment (frank-theunicorn) [inline pull-request-suggester/processor/processor.go]: Hi...🦄 So I read your PR... Meeps Meeps... this looks weird...

File: pull-request-suggester/processor/processor.go [hunk 2]
```diff
@@ -75,7 +75,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
...
+				continue
```
Comment (frank-theunicorn) [inline pull-request-suggester/processor/processor.go]: Hrmmm... Something looks weird here. Are you sure about this? 🌈

File: pull-request-suggester/processor/processor.go [hunk 3]
```diff
@@ -75,7 +75,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
...
+			}
```
Comment (frank-theunicorn) [inline pull-request-suggester/processor/processor.go]: Hi...🦄 So I read your PR... Something looks weird here. Are you sure about this? 🌈

File: pull-request-suggester/processor/processor.go [hunk 4]
```diff
@@ -75,7 +75,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
...
+
```
Comment (frank-theunicorn) [inline pull-request-suggester/processor/processor.go]: Nit... but... Something looks weird here. Are you sure about this? 🌈

File: pull-request-suggester/processor/processor.go [hunk 5]
```diff
@@ -99,6 +107,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
 
 }
 
+func in(strs []string, str string) bool {
+	for _, s := range strs {
+		if s == str {
+			return true
+		}
```
Comment (frank-theunicorn) [inline pull-request-suggester/processor/processor.go]: Henlo friend. 🦄 Something looks weird here. Are you sure about this? 🌈

File: pull-request-suggester/processor/processor.go [hunk 6]
```diff
@@ -99,6 +107,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
...
+	}
```
Comment (frank-theunicorn) [inline pull-request-suggester/processor/processor.go]: Henlo friend. 🦄 Hrmm.. this looks funny...

File: pull-request-suggester/processor/processor.go [hunk 7]
```diff
@@ -99,6 +107,15 @@ func StartConcurrentProcessorClient(opt *ClientOptions) error {
...
+	return false
+}
```
Comment (frank-theunicorn) [inline pull-request-suggester/processor/processor.go]: Me again, just one more nit...🌈 Wondering why this is here...? 🌈🐼

File: pull-request-suggester/manifest/server.yaml [hunk 8]
```diff
@@ -37,7 +37,7 @@ spec:
           ports:
             - containerPort: 80
           command: ["/go/bin/prs"]
-          args: ["--hostname", "model-server", "--port", "777", "--username", "frank-theunicorn", "--password", "tigeyboo1"]
+          args: ["--hostname", "model-server", "--port", "777", "--username", "frank-theunicorn", "--password", "tigeyboo2"]
```
Comment (kris-nova) [inline pull-request-suggester/manifest/server.yaml]: really @frank-theunicorn ?

Comment (kris-nova) [reply pull-request-suggester/manifest/server.yaml]: you didn't even catch this?

**Pull request merged**